remote_attestation/java/src/com/google/oak/remote_attestation/ClientAttestationEngine.java

```diff
@@ -30,9 +32,6 @@
 import java.util.Random;
 import oak.remote_attestation.AttestationInfo;
 import oak.remote_attestation.AttestationReport;
-import oak.remote_attestation.ClientHello;
-import oak.remote_attestation.ClientIdentity;
-import oak.remote_attestation.ServerIdentity;
 
 enum AttestationState {
```

Comment:
I think the state would be better expressed as just the expected id of the next frame.